README.md

```diff
@@ -247,6 +247,12 @@ the following command:
 $ docker run -p 4140:4140 -p 9990:9990 -v /absolute/path/to/myapp:/myapp buoyantio/linkerd:0.0.10-SNAPSHOT /myapp/linkerd.yml
 ```
 
+We supply a config that routes to a single backend on localhost:8080 for your convenience.
+
+```
+$ docker run -p 4140:4140 -p 9990:9990 -v /path/to/linkerd/linkerd/examples:/config buoyantio/linkerd:0.7.1-SNAPSHOT /config/static_namer.yaml
```

Comment:
I find this a little bit misleading to appear in the Docker section, since the static_namer.yaml isn't actually "supplied" as part of the image -- it still needs to be volume mounted.  That makes it no different from any of the other example config files provided in `linkerd/examples/`.  In fact, without docker, there's a much more straightforward way to run this config, as follows:

```
$ ./sbt linkerd-examples/static_namer:run
```